lib/index.js

```diff
@@ -41,6 +41,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
 });
 
 /**
+ * Check for the existence of an identity.
+ *
+ * @param id the ID of the identity to check.
+ * @param callback(err, exists) called once the operation completes.
+ */
+api.exists = function(id, callback) {
+  database.collections.identity.find({
```

Comment:
@dlongley sounds fine, I do want to make sure you saw this performance analysis: https://github.com/digitalbazaar/bedrock-consumer/issues/5#issuecomment-259704946